controller/src/inruptController.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller } from "./classes/Controller";
+import { InruptPermissionManager } from "./classes/permissionManager/InruptPermissionManager";
+import { InruptStore } from "./classes/stores/InruptStore";
+import { PublicResolver } from "./classes/subjectResolvers/Public";
+import { WebIdResolver } from "./classes/subjectResolvers/WebId";
+
+// const inruptController = new AccessManagement<{
+//     webId: WebIdSubject,
+//     public: PublicSubject,
+// }>(
+//     new InruptStore(),
+//     {
+//         webId: new WebIdResolver(),
+//         public: new PublicResolver(),
+//     },
+//     new InruptPermissionManager(),
+// )
+
+export const inruptController = new Controller(
+    new InruptStore(),
```

Comment:
yeah, so I would expect rather something like::

```
export const inruptController = new Controller(
    new Store(),
    {
        webId: {
            resolver: new WebIdResolver(),
            manager: new InruptPermissionManager.webIdManager()
        }, 
        public: {[same for other types]}
    })
```

Reply:
So the `webIdManager`should include each function the current `IPermissionManager` interface defines? Or did you have anything else in mind?

Reply:
Yes, because the WebIdResolver might use a different location to lookup/store the permissions (eg ACL files) than the qualifierResolver (eg ACP files).

Reply:
right now it's assumed that one permissionmanager will handle all permission types, but in reality it'll be 2-3 permissionmanagers, each handling their own (disjoint) set of permission types